.pylintrc

```diff
@@ -1,55 +1,124 @@
 # SPDX-FileCopyrightText: 2018 Free Software Foundation Europe e.V. <https://fsfe.org>
+# SPDX-FileCopyrightText: 2022 Florian Snow <florian@familysnow.net>
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 [MASTER]
-
 jobs=0
 
+
 [MESSAGES CONTROL]
 
-# Disable the message, report, category or checker with the given id(s). You
-# can either give multiple identifiers separated by comma (,) or put this
-# option multiple times (only on the command line, not in the configuration
-# file where it should appear only once).You can also use "--disable=all" to
-# disable everything first and then reenable specific checks. For example, if
-# you want to run only the similarities checker, you can use "--disable=all
-# --enable=similarities". If you want to run only the classes checker, but have
-# no Warning level messages displayed, use"--disable=all --enable=classes
-# --disable=W"
-disable=redefined-builtin,C0330,duplicate-code,logging-format-interpolation,line-too-long,implicit-str-concat,logging-fstring-interpolation,unsubscriptable-object,inherit-non-class
+disable=R0801,
+         logging-fstring-interpolation,
+enable=useless-suppression
 
 [REPORTS]
 
-# Set the output format. Available formats are text, parseable, colorized, json
-# and msvs (visual studio).You can also give a reporter class, eg
-# mypackage.mymodule.MyReporterClass.
 output-format=text
-
-# Tells whether to display a full report or only the messages
 reports=yes
 
+
 [FORMAT]
 
-# Expected format of line ending, e.g. empty (any line ending), LF or CRLF.
 expected-line-ending-format=LF
 
-# String used as indentation unit. This is usually "    " (4 spaces) or "\t" (1
-# tab).
+ignore-long-lines=^\s*(# )?<?https?://\S+>?$|^# SPDX-.*$
+indent-after-paren=4
 indent-string='    '
+max-line-length=80
```

Comment:
I believe this is set to 79 in pyproject.toml.

Reply:
Good catch!  Fixed in 5f2e9d67cdfa27ff3c460fc6f0c3f7e79ff14927.